toptal/picasso
# feat(Pagination): [FX-2059] update pagination to the latest design

[FX-2059](https://toptal-core.atlassian.net/browse/FX-2059)

### Description

- Added `variant` and `siblingCount` props.
  - Overriding button color [was discussed with the BASE team](https://toptal-core.slack.com/archives/CCC3GP6CC/p1630916998026800).
- Decrease spacing.
- Changed active page button color.

### How to test

1. Visit the temploy.
2. Check `Pagination` docs.
3. Compare to the designs: https://app.abstract.com/projects/1b06c884-06af-482a-bf12-a82f521a19a1/branches/master/commits/312f8be305c3d8c6c778ecb7bce0302eab1f03e5/files/e163c97f-163c-483d-904d-a4d279b01566/layers/0344789B-B331-4E30-8B06-03989D1DF742?collectionId=a80f780a-4a13-4052-bae9-d920ced900a9&collectionLayerId=090b6339-904f-4cd9-b77e-7aff15f93638&mode=build&selected=4224037875-46FC1419-DAD0-43DE-BD27-373E348392EB

### Screenshots

| Before.                                 | After.                                  |
| --------------------------------------- | --------------------------------------- |
|<img width="461" alt="Screen Shot 2021-09-06 at 15 44 33" src="https://user-images.githubusercontent.com/17337276/132197323-a67eb87b-508c-4212-971b-ee590c00d5d3.png">|<img width="729" alt="Screen Shot 2021-09-06 at 15 46 26" src="https://user-images.githubusercontent.com/17337276/132197335-679ee7a7-8eb8-45db-bdc1-ef6187bbd1cd.png">|


### Review


<details>
<summary>PR commands</summary>
<br />

List of available commands:

- `@toptal-bot run all` - Run whole pipeline
- `@toptal-bot run build` - Check build
- `@toptal-bot run visual` - Run visual tests
- `@toptal-bot run deploy:documentation` - Deploy documentation
- `@toptal-bot run package:alpha-release` - Release alpha version

</details>
